src/App.tsx

```diff
@@ -1,7 +1,18 @@
-import Dashboard from './pages/dashboard';
+// import Weather from './components/Weather';
+// import Dashboard from './pages/dashboard';
+// import { IWeather } from './interfaces/IWeather';
+// import { useState, useEffect } from 'react';
+// import axios, { AxiosResponse } from 'axios';
+// import { weatherData } from './services/Api';
+import React from 'react';
+import Weather from './components/Weather';
 
 const App = () => {
-  return <Dashboard />;
+  return (
+    <React.Fragment>
```

Comment:
Unnecessary `<React.Fragment>` 

in this situation I would replace it with `Container` from Mantine